README.md

```diff
@@ -14,21 +14,20 @@ Takeaway Challenge
 
```

Comment:
Nice ReadMe file - perhaps include snapshot of code?
See https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md for example